### PR TITLE
clang: Apply clang-format

### PR DESCRIPTION
**Pull request by lexnv**
The code base contains multiple coding styles,
which are hard to follow.

To ensure the same coding style as aws-c-common,
clang formatter is applied.

Signed-off-by: Alexandru Vasile <lexnv@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by axlprv — APPROVED** [review]

File: include/aws/nitro_enclaves/kms.h [hunk 1]
```diff
@@ -940,7 +940,8 @@ int aws_kms_encrypt_blocking(
  * @param[in]   key_spec     The spec of key to generate: an AES128 or an AES256 key.
  * @param[out]  plaintext    The plaintext output of the call. Should be an empty, but non-null aws_byte_buf.
  * @param[out]  ciphertext_blob The ciphertext blob output of the call. Should be an empty, but non-null aws_byte_buf.
- * @return                   Returns AWS_OP_SUCCESS if the call succeeds and plaintext and ciphertext_blob are populated.
+ * @return                   Returns AWS_OP_SUCCESS if the call succeeds and plaintext and ciphertext_blob are
+ * populated.
```
Comment (andraprs) [inline include/aws/nitro_enclaves/kms.h:944]: "populated" should be under "Returns".

**Review by andraprs — APPROVED** [review]

File: include/aws/nitro_enclaves/kms.h (hunk 1 above)
Comment (lexnv) [reply include/aws/nitro_enclaves/kms.h]: Fixed in latest revision. Thanks

**Review by andraprs — APPROVED** [review]

**Review by axlprv — APPROVED** [review]

**Review by ddraga — APPROVED** [review]

**Review by petreeftime — APPROVED** [review]
